autogtp/README.md

```diff
@@ -11,14 +11,18 @@ the SGF and training data at the end of the game.
 * curl
 * gzip and gunzip
 
-## Example of compiling - Ubuntu
+## Compiling - Ubuntu and similar
 
-    sudo apt install qt5-default qt5-qmake curl
-    qmake -qt5
-    make
+See : [Example of compiling (Ubuntu and similar)](https://github.com/wonderingabout/leela-zero/tree/minor-clearer-next#example-of-compiling---ubuntu--similar)
```

Comment:
This links to the wrong repo. I also don't understand why you change these instructions. If you want to compile autogtp only they are still OK. It's just that we do it by default when doing a full build?

Reply:
@gcp 
ok i will update the links to gcp/next

the main idea of this change was to put all commands in the same readme page (instead of splitting them between main readme and autogtp readme(that is not linked anymore in the main readme btw))
for someone who tries to compile leelaz and autogtp with little knowledge, i think it will be less confusing like that, and simpler as well
the autogtp readme will serve as documentation, and the commands will be linked to the more noticeable main readme

however, we can indeed split these 2 separate cases and leave the choice to the user


we have 2 options here : 

- option 1 : put all dependencies in the main page readme page in the same command as i did

- option 2 : still in main page readme, put a "if you also want to compile autogtp, install as well qt5 dependency"
option 2 would be something like this, for example for ubuntu and similar : 


```
    # Install build depedencies
    sudo apt install cmake libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev curl
    
    # Optionnal : if you intend to use autogtp to contribute, install autogtp dependencies as well :
    sudo apt install qt5-default qt5-qmake
```

personally, i find it to look better than my previous submit

Reply:
after thinking it again, i can see many cases where one person may want to compile only autogtp binary without leelaz

did an attemptive revert,
can you review again @gcp  ?